diagnostics/protobuf_visualize.py

```diff
@@ -0,0 +1,163 @@
+from __future__ import print_function
```

Comment:
Add copyright header.

Reply:
Also - consider renaming this file to something like tf_graph_viewer (as `protobuf` is a serialization format used by TF and this file is very specifically meant for that (and cannot be used to view other protobuf serialized files)

Reply:
added header, and changed file name